protocols/peer/contracts/Peer.sol

```diff
@@ -50,10 +56,59 @@ contract Peer is IPeer, Ownable {
   ) public {
     swapContract = ISwap(_swapContract);
     if (_peerContractOwner != address(0)) {
-      transferOwnership(_peerContractOwner);
+      whitelist[_peerContractOwner] = true;
+      super.transferOwnership(_peerContractOwner);
     }
   }
 
+  /**
+    * @notice determines if an address to interact with this peer
+    * @param addressToCheck the address to check if whitelisted or not 
+    */
+  function isWhitelisted(address addressToCheck) external returns(bool) {
```

Comment:
Insert a space after `returns`

Reply:
All of the new functions you've added do not have `_` on each parameter. This is inconsistent with the rest of the contract.